tests/run-make/asm-incr-cache/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ only-x86_64-unknown-linux-gnu
+
+extern crate run_make_support;
+
+use run_make_support::{rustc, tmp_dir};
+
+fn main() {
+    let inc_dir = tmp_dir().join("asm-incr-cache");
+
+    rustc()
+        .input("lib.rs")
+        .arg("--emit=obj,asm,dep-info,link,mir,llvm-ir,llvm-bc")
+        .arg("--crate-type=lib")
+        .arg(format!("-Cincremental={inc_dir:?})").as_str())
+        .run();
+
+    {
+        let lib = std::fs::File::options().write(true).open("lib.rs").expect("No lib.rs");
+        lib.set_modified(std::time::SystemTime::now());
```

Comment:
Rustc doesn't look at the mtime. Only cargo does.

Reply:
Good to know. I only copied what existing test did (along with deprecated Makefile) and it was necessary to replicate the bug using cargo.